scheduler/java/com/twosigma/cook/kubernetes/RemoveFinalizerHelper.java

```diff
@@ -0,0 +1,53 @@
+package com.twosigma.cook.kubernetes;
+
+import io.kubernetes.client.custom.V1Patch;
+import io.kubernetes.client.openapi.ApiClient;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.apis.CoreV1Api;
+import io.kubernetes.client.openapi.models.V1Pod;
+import io.kubernetes.client.util.PatchUtils;
+import org.joda.time.DateTime;
+
+import java.util.List;
+
+public class RemoveFinalizerHelper {
+    /** A finalizer that is attached to a pod to ensure that it is not GC'ed by K8s before cook
+     * has had a chance to collect the completion result (success or failed) */
+    static public final String collectResultsFinalizer = "ts.cook/prevent-pod-gc";
```

Comment:
We can name it anything; I'm happy to change it to any name you suggest. it should be something cook or 'twosigma.cook/....' to prevent name collision.  I'd prefer not configurable becase if you ever change it, you've got a messy migration problem. There's also no need to change it and having something greppable is valuable.

Reply:
Makes sense, how about something like `cook/process-pod`?

Reply:
I'm fine with the current name (`cook/prevent-pod-gc`) too.

Reply:
Lets keep the current name then.